isis/src/tgo/apps/tgocassis2isis/TgoCassisMapping.trn

```diff
@@ -0,0 +1,196 @@
+# End_Group
+#
+# <Array_2D_Image>
+#   <Axis_Array>
+#     <axis_name>Line</axis_name>
+#     <sequence_number>1</sequence_number>
+#     <elements>2048</elements>
+#   </Axis_Array>
+#   <Axis_Array>
+#     <axis_name>Sample</axis_name>
+#     <sequence_number>2</sequence_number>
+#     <elements>279</elements>
+#   </Axis_Array>
+# </Array_2D_Image>
+#
+# 
+# history 2018-08-09 Jeannie Backer - Changed group name from MapProjection
+#                        to ProjectionName.
+
+Group = ProjectionName
+  Auto
+  InputPosition        = (Observation_Area, Discipline_Area, cart:Cartography, cart:Spatial_Reference_Information, cart:Horizontal_Coordinate_System_Definition, cart:Planar, cart:Map_Projection) 
+  InputKey             = cart:map_projection_name
+  OutputName           = ProjectionName
+  OutputPosition       = (Object,IsisCube,Group,Mapping)
+  Translation          = (*, *)
+End_Group
+
+Group = CenterLongitude
+  Auto
+  InputPosition        = (Observation_Area, Discipline_Area, cart:Cartography, cart:Spatial_Reference_Information, cart:Horizontal_Coordinate_System_Definition, cart:Planar, cart:Map_Projection, cart:Equirectangular) 
+  InputKey             = cart:longitude_of_central_meridian
+  OutputName           = CenterLongitude
+  OutputPosition       = (Object,IsisCube,Group,Mapping)
+  Translation          = (*, *)
+End_Group
+
+Group = CenterLatitude
+  Auto
+  InputPosition        = (Observation_Area, Discipline_Area, cart:Cartography, cart:Spatial_Reference_Information, cart:Horizontal_Coordinate_System_Definition, cart:Planar, cart:Map_Projection, cart:Equirectangular) 
+  InputKey             = cart:latitude_of_projection_origin
+  OutputName           = CenterLatitude
+  OutputPosition       = (Object,IsisCube,Group,Mapping)
+  Translation          = (*, *)
+End_Group
+
+# input has different scales for x and y, but ISIS doesn't distinguish, so use x
+Group = Scale
+  Auto
+  InputPosition        = (Observation_Area, Discipline_Area, cart:Cartography, cart:Spatial_Reference_Information, cart:Horizontal_Coordinate_System_Definition, cart:Planar, cart:Planar_Coordinate_Information, cart:Coordinate_Representation)
+  InputKey             = cart:pixel_scale_x
+  OutputName           = Scale
+  OutputPosition       = (Object,IsisCube,Group,Mapping)
+  Translation          = (*, *)
+End_Group
+
+# input has different pixel resolutions for x and y, but ISIS doesn't distinguish, so use x
+Group = PixelResolution
+  Auto
+  InputPosition        = (Observation_Area, Discipline_Area, cart:Cartography, cart:Spatial_Reference_Information, cart:Horizontal_Coordinate_System_Definition, cart:Planar, cart:Planar_Coordinate_Information, cart:Coordinate_Representation)
+  InputKey             = cart:pixel_resolution_x
+  OutputName           = PixelResolution
+  OutputPosition       = (Object,IsisCube,Group,Mapping)
+  Translation          = (*, *)
+End_Group
+
+## for min/max lat/lons, can we assume equivalent to N/S E/W min/max ? 
```

Comment:
This turned out to not be true, but was handled separately in the software. I need to update this to remove the comment but am going to wait until the Jenkins run completes